Index: venv/credential.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import socket\r\n\r\n\r\nserver_base = socket.gethostbyaddr(socket.gethostname())[0]\r\ndata_base = 'middleware'\r\nbase_webservice = \"TOTAL\"\r\nuser_webservice = \"FUSION\"\r\nsenha_webservice = \"GsiTNjR*@LuoL3\"\r\n
===================================================================
diff --git a/venv/credential.py b/venv/credential.py
--- a/venv/credential.py	
+++ b/venv/credential.py	
@@ -5,4 +5,8 @@
 data_base = 'middleware'
 base_webservice = "TOTAL"
 user_webservice = "FUSION"
+<<<<<<< HEAD
+senha_webservice = "2GsiTNjR*@LuoL3"
+=======
 senha_webservice = "GsiTNjR*@LuoL3"
+>>>>>>> parent of b99d284 (Diversos)
